repo/packages/M/mongodb-replicaset/0/command.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
As of packagingVersion 3.0, command.json is deprecated in favor of the .cli property of resource.json